<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="CA-LCV0Q8Y2" failures="1" tests="2" name="1" time="52.445" errors="0" timestamp="2022-10-04T16:12:39 EDT">
  <testcase classname="tests.TestExample2" name="UID1111_testExample2" time="0.737"/>
  <testcase classname="tests.TestExample2" name="UID1112_testExample22" time="0.048">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at com.assertion.BaseAssertion.shouldBeEqual(BaseAssertion.java:143)
at com.assertion.BaseAssertion.shouldBeEqual(BaseAssertion.java:116)
at tests.TestExample2.UID1112_testExample22(TestExample2.java:77)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- UID1112_testExample22 -->
</testsuite> <!-- 1 -->
